packages/shared/models/user.js

```diff
@@ -0,0 +1,24 @@
+import mongoose from 'mongoose'
+
+const userSchema = new mongoose.Schema({
+  id: String,
+  username: String,
+  passwordHash: String,
+  avatarURL: String,
+  dashboards: [
+    {
+      id: String,
+      captionURL: String,
```

Comment:
change to imageCoverId: Number

Reply:
but component ImagePicker returns url to unsplash photo.